tasks/cycles/cycles.js

```diff
@@ -29,11 +55,18 @@ const personIncomes = {
   otherExpences: -300
 };
 
+totalIncome=0;
+ArrayOfValues = Object.values(personIncomes);
+for (let i = 0; i < Object.values(personIncomes).length; i++){
```

Comment:
JFYI

Also, your could use here `for in`
`
for (let el in personIncomes) {
  totalIncome += (personIncomes[el]);
};

`